README.md

```diff
@@ -608,6 +608,14 @@ Absolute path to the OpenSSH User CA Certificate (TrustedUserCAKeys) for use wit
 
 - *Default*: undefined
 
+sshd_config_authorized_principals_file
+--------------------------------------
+String path (relative or absolute) to the `authorized_principals` file. Sets the `AuthorizedPrincipalsFile` setting in `sshd_config`
```

Comment:
No, because the default setting is not an absolute path and users may choose to not use an absolute path. Not to mention that value can also take tokens (which for our purposes can just be part of the string)